### ISSUE TITLE
[GH Request] @openedx/openedx-demo-course-maintainers group

### ISSUE THREAD

**Issue by sarina** [openedx/axim-engineering#969]
### Firm Name

Axim

### Urgency

Low (2 weeks)

### Requested Change

Add group @openedx/openedx-demo-course-maintainers and add Sarina + John Swope to it

### Reasoning

We're maintaining the demo course!

**Comment by sarina** [comment]
made https://github.com/orgs/openedx/teams/openedx-demo-course-maintainers blocked on finding out John's GH username

**Comment by sarina** [comment]
Closing in favor of #971
